src/Aspire.Cli/Program.cs

```diff
@@ -111,8 +113,32 @@ private static FileInfo UseOrFindAppHostProjectFile(FileInfo? projectFile)
         }
 
         var projectFilePaths = Directory.GetFiles(Environment.CurrentDirectory, "*.csproj");
-        var projectFilePath = projectFilePaths.Single();
-        return new FileInfo(projectFilePath);
+        try 
+        {
+            var projectFilePath = projectFilePaths?.Single();
+            if (projectFilePath is null)
+            {
+                throw new InvalidOperationException("No project file found.");
```

Comment:
I would suggest not using exceptions for control flow. Just write to the console here.

Reply:
this was a hack because it was throwing before it could get to the point where it writes to the console... @mitchdenny will fix it lol